src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
+import deepClone from '../../helpers/json/deepClone';
+import StoryPromo from '.';
+
+let item;
+const compleateItem = {
+  headlines: {
+    headline: 'A headline',
+  },
+  summary: 'Summary text',
+  timestamp: 1556795033,
+  indexImage: {
+    path: '/cpsprodpb/0A06/production/image.jpg',
+    height: 1152,
+    width: 2048,
+    altText: 'Image Alt text',
+  },
+};
+
+describe('StoryPromo Container', () => {
+  shouldMatchSnapshot(
+    'should render correctly',
+    <StoryPromo item={compleateItem} />,
+  );
+
+  describe('assertion tests', () => {
+    beforeEach(() => {
+      item = deepClone(compleateItem);
+    });
+
+    it('should correctly include expected elements', () => {
+      const { container } = render(<StoryPromo item={item} />);
+
+      expect(container.getElementsByTagName('h3')[0].innerHTML).toEqual(
+        item.headlines.headline,
+      );
+      expect(container.getElementsByTagName('p')[0].innerHTML).toEqual(
+        item.summary,
+      );
+      expect(container.getElementsByTagName('time')[0].innerHTML).toEqual(
+        '2 May 2019',
+      );
+      expect(container.getElementsByTagName('img')[0].src).toEqual(
+        `https://ichef.bbci.co.uk/news/660${item.indexImage.path}`,
+      );
+      expect(
+        container.getElementsByTagName('img')[0].getAttribute('alt'),
+      ).toEqual(item.indexImage.altText);
+    });
+
```

Comment:
Please also add a test to assert that the `amp-img` variant is being rendered when `platform` is `amp`.

Reply:
Done :)